src/components/Form/Location/Location.jsx

```diff
@@ -868,6 +869,7 @@ Location.defaultProps = {
   addressBooks: {},
   addressBook: '',
   isPoBoxAllowed: true,
+  showPostOffice: true,
```

Comment:
Does this line addition enable APO addresses for all locations by default?

Reply:
Yes it does, but that's a mistake. It should be `false` as default.

It should be `false` because in `src/components/Form/Location/Address.jsx` line 585, it's `false` by default.